New Node Progress

josephkane opened the pull request:
* show task logs and steps for nodes added to a cluster

* fix for https://github.com/supergiant/control/issues/1103

nikage commented on cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts:
```diff
@@ -269,9 +268,42 @@ export class ClusterComponent implements OnInit, OnDestroy {
+
+      this.getKubeStatus(this.clusterId).subscribe(
+        tasks => {
+          let executing = tasks.filter(t => t.status == "executing");
```
this looks like non rxjs way(

nikage commented on cmd/ui/assets/src/app/clusters/cluster/cluster.component.ts:
```diff
@@ -464,6 +496,7 @@ export class ClusterComponent implements OnInit, OnDestroy {
   }
 
   expandRow = (_, row) => row.hasOwnProperty('detailRow');
+  // expandRow = (_, row) => { console.log("WHEN: ", row); return row.hasOwnProperty('detailRow') };
```
pls remove commented code

josephkane replied:
thanks, good catch

eldridgeh commented:
@josephkane 
1. Once the node is provisioned the provisioning section does not go away. I had to refresh the page. This happens in DO and AWS.
<img width="1289" alt="screen shot 2018-12-05 at 12 25 43 pm" src="https://user-images.githubusercontent.com/35240784/49535103-ebcb2580-f888-11e8-8b8d-72c6456c1b81.png">
2. It takes a few seconds for the provisioning section to show up when adding a node to a DO. Would it be possible to make it show up faster? This looked to be instant with AWS
3. I created a cluster in AWS and when it switched over to the cluster details page, the provisioning node table showed up showing that the only node in the cluster was provisioning. We should only see the provisioning table when adding a node. It did not do this with DO. 
4. This is more than likely related to 1, after deleting a node the provisioning table did not go away.

josephkane commented:
@eldridgeh 
1. great catch, fixed
2. This seem to be a symptom of A.) no "planned" for DO nodes or B.) DO node objects aren't created at the same time as AWS nodes are in the DB. This one is sort of out of my hands (I could be mistaken)
3. This is most likely due to the cluster being declared "operational" before the last node is finished provisioning. In that case, this table would work as designed (because that node actually is still provisioning)
4. fixed with 1
